build.gradle

```diff
@@ -224,6 +246,23 @@ tasks.named("dependencyUpdates").configure {
 
 gradle.taskGraph.whenReady { graph ->
     def tasks = gradle.startParameter.taskNames.toString()
+
+    def hasRootUserManagementTask = tasks.contains(":user-management:bootWar") || tasks.contains(":user-management:executeTests") || tasks.contains(":user-management:jacocoTestReport")
+        || tasks.contains(":user-management:spotlessCheck")  || tasks.contains(":user-management:spotlessApply")  || tasks.contains(":user-management:dependencyUpdates")
+        || tasks.contains(":user-management:checkstyleMain")  || tasks.contains(":user-management:jacocoTestCoverageVerification")  || tasks.contains(":user-management:test")
+        || tasks.contains(":user-management:bootRun") || tasks.contains(":user-management:buildJarForDocker") || tasks.contains(":user-management:cleanResources")
+        || tasks.contains(":user-management:bootJar")
```

Comment:
Couldn't this be simplified to `task.contains(":user-management:")`? Technically, you would allow other tasks than the above-defined but I'm not sure if it wins the tradeoff with readability.

If you simplify it, the same can be applied to the gateway tasks. Either here or in the other PR. I'd suggest the other one until it's ready to merge.

Reply:
It works, I don't know why I haven't done it like that from the beginning :D I changed it for the gateway as well